### PR TITLE
Bump prettier and update linting

### PR DESCRIPTION
**Pull request by odinuge**
Now Prettier also formats markdown :rocket:

### CODE REVIEW

**Comment by odinuge** [comment]
When using the eslint prettier addon, it only checks the js files. Should we just remove it, or run `lint:pretteier` on only css and md? It may be useful for people using eslint with vim etc., but with format on autosave, it shouldn't be necessary...

File: package.json [hunk 1]
```diff
@@ -15,10 +15,11 @@
     "test": "NODE_ENV=test jest",
     "test:coverage": "yarn run test -- --coverage",
     "test:watch": "yarn run test -- --watch",
-    "lint": "yarn run lint:js && yarn run lint:css",
+    "lint": "yarn run lint:js && yarn run lint:css && yarn run lint:prettier",
```
Comment (ekmartin) [inline package.json]: isn't this already done in `lint:js` with eslint-plugin-prettier? could probably remove that then?

Comment (ekmartin) [reply package.json]: lol 2slow, yeah, just remove eslint-plugin-prettier :)

**Comment by odinuge** [comment]
Guess I should update `README` too

**Pull request merged**